libs/git_wrapper/Git_wrapper.mli

```diff
@@ -67,7 +67,7 @@ val get_project_url : unit -> string option
     [git ls-remote] or from the [.git/config] file. It returns [None] if it
     found nothing relevant. *)
 
-val get_git_logs : unit -> string list
+val get_git_logs : ?since:Common2.float_time option -> unit -> string list
```

Comment:
then you need to update the comment saying if no since then it defaults to the last 30 days.

Reply:
and then I'd rather be explicit at the calling site in the dump_contributions() function and still pass a ~since:"last 30 days", even if it's the default, so it's clear in the semgrep code (this libs/ code is supposed to contain general libraries that can be used outside semgrep) that we use a 30 days (and maybe even add a link to the semgrep.dev/docs/usage-limit to explain why we use 30 days in dump_contributions().

Reply:
Yeah that makes sense. Updating!

Reply:
Do you think we need to propagate the `since` param all the way up to Core_CLI?

Reply:
no, you can just add the ~since:"30 days" in the call to get_git_logs in Parse_contribution.ml
(and maybe add a comment about it in Parse_contribution.mli, and the link to semgrep.dev/usage-limit there)

Reply:
Parse_contributions.ml is part of semgrep (it's not in libs/, it's in src/), and its objective is to parse the contributions for semgrep, for the app, so it's perfectly fine to hardcode the 30 days there;